crates/hyperswitch_connectors/src/connectors/trustpayments/transformers.rs

```diff
@@ -345,69 +388,269 @@ impl TryFrom<&TrustpaymentsRouterData<&PaymentsAuthorizeRouterData>>
+                    sitereference: auth.site_reference.clone().expose(),
+                    requesttypedescriptions: vec!["AUTH".to_string()],
+                    accounttypedescription: "ECOM".to_string(),
+                    currencyiso3a: item.router_data.request.currency.to_string(),
+                    baseamount: item.amount.clone(),
+                    orderreference: item.router_data.connector_request_reference_id.clone(),
+                    errorurlredirect: return_url.clone(),
+                    successfulurlredirect: return_url.clone(),
+                    billingdata: Some(billing_details.clone()),
+                    ..Default::default()
+                }],
+            }),
+            BankRedirectData::Trustly { .. } => Ok(Self {
+                alias: auth.username.clone().expose(),
+                version: TRUSTPAYMENTS_API_VERSION.to_string(),
+                request: vec![TrustpaymentsPaymentRequestData {
+                    paymenttypedescription: Some(TrustpaymentsPaymentTypes::TRUSTLY),
+                    sitereference: auth.site_reference.clone().expose(),
+                    requesttypedescriptions: vec!["AUTH".to_string()],
+                    accounttypedescription: "ECOM".to_string(),
+                    currencyiso3a: item.router_data.request.currency.to_string(),
+                    baseamount: item.amount.clone(),
+                    orderreference: item.router_data.connector_request_reference_id.clone(),
+                    returnurl: return_url.clone(),
+                    billingdata: Some(billing_details.clone()),
+                    ..Default::default()
+                }],
+            }),
+            _ => Err(errors::ConnectorError::NotImplemented(
+                "Bank redirect method not supported".to_string(),
+            )
+            .into()),
+        }
+    }
+}
+
+impl
+    TryFrom<(
+        &TrustpaymentsRouterData<&PaymentsAuthorizeRouterData>,
+        &TrustpaymentsAuthType,
+        &Card,
+    )> for TrustpaymentsPaymentsRequest
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+
+    fn try_from(
+        (item, auth, req_data): (
+            &TrustpaymentsRouterData<&PaymentsAuthorizeRouterData>,
+            &TrustpaymentsAuthType,
+            &Card,
+        ),
+    ) -> Result<Self, Self::Error> {
+        let request_types = match item.router_data.request.capture_method {
+            Some(common_enums::CaptureMethod::Automatic) | None => vec!["AUTH".to_string()],
+            Some(common_enums::CaptureMethod::Manual) => vec!["AUTH".to_string()],
+            Some(common_enums::CaptureMethod::ManualMultiple)
+            | Some(common_enums::CaptureMethod::Scheduled)
+            | Some(common_enums::CaptureMethod::SequentialAutomatic) => {
+                return Err(errors::ConnectorError::NotSupported {
+                    message: "Capture method not supported by TrustPayments".to_string(),
```

Comment:
```suggestion
                    message: "Capture method not supported".to_string(),
```
The connector is mentioned bellow `connector: "TrustPayments",`

Reply:
removed "by TrustPayments" from the error message